_docs/geyser/portforwarding.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Portforwarding
+---
+
+# Portforwarding
+This page contains information on how to set up portforwarding required for Geyser to work when self-hosting.
+There are also guides for specific configurations, such as Docker/Pterodactyl, or on specific VPS/KVM providers, such as OVH or Oracle Cloud. 
+
+<div class="alert alert-warning" role="alert">
+	If you are using a Minecraft server hosting provider (e.g. Aternos, or Apex Hosting), you should refer to the hosting provider setup on the <a href="/geyser/setup/">setup</a> page instead.
+</div>
+
+## Portforwarding on Linux/Windows/macOS
+To allow others to play on your server, you will need to set up portforwarding on the device hosting Geyser.
+Additionally, you will need to portforward the port on your router/modem if you want the server to be accessible outside your own home network. 
+See [here](https://www.howtogeek.com/66214/how-to-forward-ports-on-your-router/) or [here](https://www.lifewire.com/how-to-port-forward-4163829) for helpful guides.
+Do note: if you do not have a static IP address, your IP address may change over time. 
+
+<div class="alert alert-info" role="alert">
+	Some ISPs (Internet Service Providers) block certain ports, or don't allow you to open ports (e.g. by using CGNAT, which doesn't allow you to open a port with a dynamic IP).
+Other ISPs may require you to pay extra for a static IP address. <br>
+As an alternative to portforwarding, you can use <a href="/geyser/playit-gg/">playit.gg</a> to create a tunnel.
+</div>
+
+### Windows
+To open a port on Windows, you will need to open the port through the Windows Firewall. There are multiple ways to do this:
+
+**Powershell** <br>
+To open a port on UDP (in our example, port 19132), run the following command in an administrator Powershell: <br>
+`New-NetFirewallRule -DisplayName "Geyser" -Direction Inbound -Protocol UDP -LocalPort 19132 -Action Allow` <br>
+After running this command, a rule named "Geyser" was created to allow UDP traffic on port 19132. <br>
+
+**Windows Defender Firewall with Advanced Security** <br>
+1. Search for "Windows Defender Firewall with Advanced Security" in the start menu, and open it. [Image](https://cdn.discordapp.com/attachments/613194762249437245/1139289055612370964/image.png)
```

Comment:
Can we move these images to part of the repo just incase the files are removed from discord? (whole document)

Reply:
Done, created an /img/guides/port-forwarding folder for these